bundles/framework/coordinatetool/plugin/CoordinateToolPlugin.js

```diff
@@ -182,12 +182,12 @@ Oskari.clazz.define('Oskari.mapframework.bundle.coordinatetool.plugin.Coordinate
         _showPopup: function() {
             var me = this,
                 loc = me._locale,
-                popupTitle = loc.popup.title,
+                popupTitle = loc('display.popup.title'),
                 popupContent = me._templates.popupContent.clone(),
                 crs = me.getMapModule().getProjection(),
-                crsDefaultText = loc.crs.default,
+                crsDefaultText = loc('display.crs.default'),
                 popupName = 'xytoolpopup',
-                crsText = loc.crs[crs] || crsDefaultText.replace('{crs}', crs),
+                crsText = loc('display.crs.'+crs) || crsDefaultText.replace('{crs}', crs),
```

Comment:
getMsg() allows built-in templating/interpolation so the replace call could be removed with something like this:

      crsDefaultText = loc('display.crs.default', {crs: crs});